UI/ui-validation.cpp

```diff
@@ -117,3 +117,47 @@ UIValidation::StreamSettingsConfirmation(QWidget *parent, OBSService service)
 
 	return StreamSettingsAction::ContinueStream;
 }
+
+StreamSettingsAction UIValidation::BandwidthModeConfirmation(QWidget *parent,
+							     OBSService service)
+{
+	obs_data_t *settings = obs_service_get_settings(service);
+	bool bwtest = obs_data_get_bool(settings, "bwtest");
+	obs_data_release(settings);
+
+	if (bwtest && parent->isVisible()) {
+
+		QMessageBox messageBox(parent);
+		messageBox.setWindowTitle(QTStr("ConfirmBWTest.Title"));
+		messageBox.setText(QTStr("ConfirmBWTest.Text"));
+
+		QPushButton *cancel;
+		QPushButton *settings;
+		QPushButton *stream;
+
+#ifdef __APPLE__
+#define ACCEPT_BUTTON QMessageBox::AcceptRole
+#define REJECT_BUTTON QMessageBox::ResetRole
+#else
+#define ACCEPT_BUTTON QMessageBox::NoRole
+#define REJECT_BUTTON QMessageBox::NoRole
+#endif
```

Comment:
Is this needed?  Looks like copy paste from above code that already defines these. 
Either remove or consolidate&foist these to the top of the file.